2nd Battlegroup/ACLF to NGR outfit updates

Apogee-dev opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request

Repaths various clothing items from the ACLF Grunt and 2nd Battlegroup Captain outfits to more appropriately reflect their connection to the New Gorlex Republic.

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game

local lorema DEMANDS lore compliance, dozens wounded

## Changelog

:cl:
tweak: Tweaked Syndicate Grunt/Captain clothing descriptions for lore compliance
refactor: changed paths for ACLF Grunt and 2nd Battlegroup outfit parts
fix: 2nd Battlegroup captain jacket can now hold firearms
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Apogee-dev committed:
repaths ACLF grunt and officer jumpsuits

Updates the path & desc of ACLF Grunt and 2nd BG Captain outfit pieces to reflect NGR allegiance

Apogee-dev committed:
missed a spot

meemofcourse commented on code/modules/clothing/suits/miscellaneous.dm:
```diff
@@ -686,11 +686,11 @@
 	body_parts_covered = ARMS
 
-/obj/item/clothing/suit/gorlex
+/obj/item/clothing/suit/ngr_captain
```
could you please repath this under armor, or at least set the allowed var to the security vest's so you can actually hold weapons?

Apogee-dev replied:
I'm gonna repath it to capcarapace for consistency with the other captain jackets, but should I buff its armor? Right now even the Cybersun haori has normal carapace armor.

meemofcourse replied:
yeah, sure

Apogee-dev committed:
repathed 2BG jacket to carapace

Tweaked path and icon path for 2BG jacket as well as cybersun haori for consistency

Apogee-dev committed:
missed a spot
